src/renderer/state/setting/test/logics.test.ts

```diff
@@ -27,6 +27,7 @@ import {
 import { updateSettingsLogic } from "../logics";
 import {
   getLimsHost,
+  getShowAddAnnotationHint,
```

Comment:
looks like this can be removed (according to the build alert)?

Reply:
Ah I had added the wrong test -- now it should be used